### ISSUE TITLE
API endpoints no longer functional

### ISSUE THREAD

**Issue by infojunkie** [meedan/montage#32]
We are experiencing periodic 503 errors at the API level, e.g. `https://greenday-project.appspot.com/_ah/api/greenday/v1/users/me`. The problem is that there is no log of this error, so we are unable to find the layer that's failing.

Here's the corresponding Google Stackdriver alert: https://app.google.stackdriver.com/incidents/0.kwr6e3d3pf5y?project=greenday-project

**Comment by infojunkie** [comment]
Other people may be talking about it too: https://groups.google.com/forum/#!topic/google-appengine/Cu9s9DXNUv8

**Comment by infojunkie** [comment]
Created an issue for Google: https://issuetracker.google.com/115478933

**Comment by infojunkie** [comment]
Right now, there is no log for API access. We should add this to see the calls that cause the 503.

**Comment by infojunkie** [comment]
As per Google Support:
> I've noticed that your handlers:is still point to: - url: '/_ah/spi/.*'. If so, you're probably using Cloud Endpoints framework V1. 
> I'd like to inform you ahead that Cloud Endpoints Framework v1 is already deprecated [1]. It is also scheduled to shutdown since September 3, 2018 [2]. 
> If the information I've gathered were correct so far, kindly migrate to Cloud Endpoints framework v2 [3]. You'll notice that the url handler will be changed to /_ah/api/.*. (you currently have /_ah/spi/.*). 
> In line with the visibility of the error on the other hand, Google Cloud no longer support this specific feature and has probably removed most of the V1 logs. Since we opt our customers to migrate to V2 as soon as possible. 

tl;dr we need to migrate to Endpoints V2 https://cloud.google.com/endpoints/docs/frameworks/python/migrating

**Comment by infojunkie** [comment]
On the other hand, we've received a useful comment on the Google Issue Tracker:
> The path of the uptime check seem to be off.  This seem like a configuration issue where the path is not intended to return a response and seem to be directing to your API which would be a path accessed programmatically. The reason there is no logs seem to be as explained above, that path is not reaching your application, thus the 503 errors, which is expected.

I have removed the offeding up time checks and will monitor the app.

**Comment by infojunkie** [comment]
Upgraded Google Endpoints and deployed to live. Seems to work now.
